src/Features/LanguageServer/Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActionsSource.cs

```diff
@@ -474,34 +481,19 @@ bool IsActionAndSpanApplicable((CodeAction action, TextSpan? applicableSpan) act
         /// <see cref="UnifiedSuggestedActionSetPriority.Low"/> and should show up after fixes but before
         /// suppression fixes in the light bulb menu.
         /// </remarks>
-        private static UnifiedSuggestedActionSet OrganizeRefactorings(Workspace workspace, CodeRefactoring refactoring)
+        private static async Task<UnifiedSuggestedActionSet> OrganizeRefactoringsAsync(
+            Workspace workspace,
+            Document document,
+            TextSpan selection,
+            CodeRefactoring refactoring,
+            CancellationToken cancellationToken)
         {
             using var refactoringSuggestedActionsDisposer = ArrayBuilder<IUnifiedSuggestedAction>.GetInstance(out var refactoringSuggestedActions);
 
-            foreach (var codeAction in refactoring.CodeActions)
+            foreach (var (action, applicableToSpan) in refactoring.CodeActions)
             {
-                if (codeAction.action.NestedCodeActions.Length > 0)
```

Comment:
Code just moved into a local function `GetUnifiedSuggestedActionSetAsync` below as it needs to be invoked recursively.